src/test/java/org/springframework/hateoas/SimpleResourceAssemblerTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.hateoas;
+
+import static org.assertj.core.api.Assertions.*;
+
+import lombok.Data;
+
+import java.util.Collections;
+
+import org.junit.Test;
+
+/**
+ * @author Greg Turnquist
+ */
+public class SimpleResourceAssemblerTest {
```

Comment:
Test class naming seems to be a little inconsistent, but I think it should be `...UnitTest`.

Side note, since you are aiming for Spring Data style: that would be `...UnitTests`. Of course, that would need a change in the surefire configuration and should be done in a separate change.